src/amo/sagas/collections.js

```diff
@@ -334,10 +335,17 @@ export function* deleteCollection({
     };
     yield call(api.deleteCollection, params);
 
+    // Unload the collection from state.
+    yield put(deleteCollectionBySlug(slug));
```

Comment:
it is definitely a follow-up issue, but having an action named `deleteCollection` that calls `deleteCollectionBySlug` is super weird. I was like "oh shhhh, I missed this `deleteCollection` action when reviewing #5161" but actually, `deleteCollection` and `deleteCollectionBySlug` have different purposes.

Can we rename `deleteCollectionBySlug` to `unloadCollectionBySlug`?

Reply:
Good idea. I will file a follow-up.

Reply:
I opened https://github.com/mozilla/addons-frontend/issues/5238